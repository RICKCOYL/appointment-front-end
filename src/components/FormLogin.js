/* eslint-disable no-console */
import React from 'react';

const FormLogin = () => (
  <div id="login-box">
    <div className="left">
      <input type="text" name="email" placeholder="E-mail" />
      <input type="password" name="password" placeholder="Password" />

      <input type="submit" name="signup_submit" value="Login" />
    </div>

    <div className="right">

      <input type="submit" className="social-signin facebook" value="Log in with facebook" />
      <input type="submit" className="social-signin twitter" value="Log in with Twitter" />
      <input type="submit" id="google" className="social-signin google" value="Log in with Google" />
    </div>
    <div className="or">OR</div>
  </div>
);
export default FormLogin;
