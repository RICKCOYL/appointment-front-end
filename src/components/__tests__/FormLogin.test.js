import React from 'react';

import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import FormLogin from '../FormLogin';

configure({ adapter: new Adapter() });
const mockStore = configureMockStore();
const store = mockStore({});
const setup = () => shallow(
  <Provider store={store}>
    <FormLogin />
  </Provider>,
);

describe('Formbooking Component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });
  test('renders the booking form', () => {
    const page = component.find('.Formbooking');
    expect(page).toMatchSnapshot();
  });
});
