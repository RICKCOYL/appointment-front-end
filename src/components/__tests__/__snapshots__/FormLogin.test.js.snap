// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Formbooking Component renders the booking form 1`] = `ShallowWrapper {}`;
